packages/transactions/src/builders.ts

```diff
@@ -202,9 +204,9 @@ export async function broadcastRawTransaction(
 
   const text = await response.text();
   try {
-    return JSON.parse(text) as TxBroadcastResult;
+    return { txid: JSON.parse(text) } as TxBroadcastResult;
   } catch (e) {
-    return text;
+    return { txid: text } as TxBroadcastResult;
```

Comment:
This is an interesting error condition. Looks like it would only happen on failure to parse JSON, yet this code is only reached on a successful (2xx) http response? If so, it seems like this should throw similar to the line above like 
``throw new Error(`Failed to parse broadcasted transaction response ${e}`);``

But if there's an expected reason for this catch condition to be reached, then the error `e` should be included in the return object.

Reply:
@zone117x This scenario is already handled if the response is not ok at line 204 right after `fetchPrivate` call. The purpose of extra try catch and JSON.parse was to remove the extra double quotes that comes in api response around the txid.  Sample response is here:
```
'"9d82896f72b3af3c0ed9364b22a524597cc3ea662eaf681e4a8f55f2be2a53f3"'
```
You can see the extra double quotes inside the string. So previously JSON.parse was used. I tried to make it more cleaner using replace function to remove the extra quotes. Let me know if it makes sense.

Reply:
Ah I see, looks good to me 👍